### DISCUSSION TITLE
Is there any exploit for Chome OS 120?

### DISCUSSION THREAD

**Discussion by omgjkvyufui** [3kh0/ext-remover#897] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I need an exploit that alows me to play games on my chromebook again

**Comment by S-PScripts** [comment]
> I need an exploit that alows me to play games on my chromebook again

how did chromeos v120 ban games? that would be your school controlling with their blocker.
if you were using skiovox, use cryptosmite
holy shit did i say dekrpytionite

**Reply by DankFireR** [reply]
what is dekryptonite? just wondering

**Reply by S-PScripts** [reply]
some titanium network exploit that's suppost to be out by now

**Reply by omgjkvyufui** [reply]
whenever i use skivox i close the school tab and then there is the setting tab, i turn on my wifi and then I close that tab and then there are no other tabs for me to open google with, am I missing a step?

**Reply by legoskid** [reply]
> some titanium network exploit that's suppost to be out by now

it aint going to be out for a while

**Reply by legoskid** [reply]
> whenever i use skivox i close the school tab and then there is the setting tab, i turn on my wifi and then I close that tab and then there are no other tabs for me to open google with, am I missing a step?

Did you press the help icon

**Reply by HeroLink6** [reply]
It doesn't work on version 120, since it is only version 118 and below... but it's other methods do.

**Reply by omgjkvyufui** [reply]
on the insturction it said nothin about a help icon and when am I suposse to press the help icon?

**Reply by B06e** [reply]
what is dekryptonite?

**Reply by S-PScripts** [reply]
some titanium network exploit that's suppost to be out by now

**Reply by omgjkvyufui** [reply]
i dont think its out i have tried to search for it

**Reply by v4pl** [reply]
Skiovox  works on 120 just use new method.

**Comment by synapt0** [comment]
I believe skiovox's alternative methods for Skiovox can work on 120, but I'm unsure.

**Reply by legoskid** [reply]
Yep it can

**Reply by HeroLink6** [reply]
I think it can

**Reply by omgjkvyufui** [reply]
i tried it i dont get the unblocked tab

**Reply by CGRmags** [reply]
@omgjkvyufui it works

**Reply by omgjkvyufui** [reply]
tell me the steps?

**Reply by Gitkickva** [reply]
doesn't work for me

**Comment by legoskid** [comment]
Do esc+refresh+power and press tab, look for kernver and make sure it isn't 103, if it isn't then downgrade https://chrome100.dev/guide/ and http://cros.tech

If it is on 103 use this: https://ptb.discord.com/channels/419123358698045453/1173600989652193340/1187198006349529198

**Reply by omgjkvyufui** [reply]
the link takes me to nothing, it says i dont have acces to the channels or they dont exisist (the discord link)

**Reply by PenguinLord561** [reply]
@omgjkvyufui read what it says and look back on how stupid your reply was... you said you have chrome os 120...

**Comment by funkymokey** [comment, marked as answer]
just put up the white flag and use a proxy

**Reply by omgjkvyufui** [reply]
what white flag @funkymokey

**Reply by S-PScripts** [reply]
when he says white flag he means to surrender. he says that's there's no exploits that exist and that you should just use a proxy

**Reply by omgjkvyufui** [reply]
are there any proxies you know?

**Reply by S-PScripts** [reply]
i don't supply any proxies anymore but here's their discords https://github.com/S-PScripts/chromebook-utilities/blob/main/Prxes/What%20happened%20to%20the%20proxies%3F

**Reply by funkymokey** [reply]
you could also deploy a proxy via ultraviolet on [this](https://github.com/titaniumnetwork-dev/Ultraviolet-App), its really easy

**Reply by S-PScripts** [reply]
none of those are actually viable.
koyeb you need to put your card creditanals in, replit doesn't host for free anymore, railway takes links down quick and glitch is... just glitch

**Reply by AshtonDavies** [reply]
> none of those are actually viable.
> koyeb you need to put your card creditanals in, replit doesn't host for free anymore, railway takes links down quick and glitch is... just glitch 

Netlify or vercel are your other options.

**Reply by S-PScripts** [reply]
yes, i guess so but i think vercel blocked key words like cyclic (for example, interstellar)

**Reply by omgjkvyufui** [reply]
what if I use codespaces?

**Reply by funkymokey** [reply]
try anything, i just said that because a friend showed me how to deploy a proxy
